### PR TITLE
Modify step definition docs to provide better Ruby info.

### PR DESCRIPTION
**Pull request by marnen**

### CODE REVIEW

File: content/docs/gherkin/step-organization.md [hunk 1]
```diff
@@ -124,6 +126,6 @@ This helps in a number of ways,
 
 You can handle other behaviours, like *validating a webpage, clicking a button, etc.*, the same way.
 
-We suggest taking a look at the [Factory Design Pattern] (https://refactoring.guru/design-patterns/factory-method) as well.
+{{% text "java" %}}We suggest taking a look at the [Factory Design Pattern] (https://refactoring.guru/design-patterns/factory-method) as well.{{% /text %}}{{% text "kotlin" %}}We suggest taking a look at the [Factory Design Pattern] (https://refactoring.guru/design-patterns/factory-method) as well.{{% /text %}}
```
Comment (aslakhellesoy) [inline content/docs/gherkin/step-organization.md]: Why these conditionals? Do you not think the paragraph is relevant for all readers?

File: content/docs/gherkin/step-organization.md [hunk 2]
```diff
@@ -106,7 +106,9 @@ public void i_want_to_open_page(String webpage) {
 ```
 
 ```ruby
- //TODO
+ Given 'I go to the {string} page' do |page|
```
Comment (aslakhellesoy) [inline content/docs/gherkin/step-organization.md:109]: 👍

File: content/docs/gherkin/step-organization.md (hunk 1 above)
Comment (marnen) [reply content/docs/gherkin/step-organization.md]: Exactly. As mentioned in the commit message (which I realize isn’t that noticeable), the Factory pattern is not terribly useful in Ruby or JS. I don’t know Kotlin at all, so I left it in.

File: content/docs/gherkin/step-organization.md (hunk 2 above)
Comment (mlvandijk) [reply content/docs/gherkin/step-organization.md:109]: Thnx! (I don't know Ruby)

File: content/docs/gherkin/step-organization.md (hunk 1 above)
Comment (mlvandijk) [inline content/docs/gherkin/step-organization.md]:
```suggestion
{{% text "java,kotlin" %}}We suggest taking a look at the [Factory Design Pattern] (https://refactoring.guru/design-patterns/factory-method) as well.{{% /text %}}
```

Comment (mlvandijk) [reply content/docs/gherkin/step-organization.md]: Please use polyglot annotation for Java and Kotlin together for easier maintenance.

**Review by mlvandijk — CHANGES_REQUESTED** [review]

File: content/docs/gherkin/step-organization.md (hunk 1 above)
Comment (marnen) [reply content/docs/gherkin/step-organization.md]: I’ll be happy to! I didn’t know that syntax existed.

**Review by luke-hill — APPROVED** [review]
This gives a little bit more info. I'm aware it's not perfect. But it's better than before. So I'll merge this in.

**Comment by aslakhellesoy** [comment]
Hi @marnen,

Thanks for your making your first contribution to Cucumber, and welcome to the Cucumber committers team! You can now push directly to this repo and all other repos under the cucumber organization! 🍾

In return for this generous offer we hope you will:

* ✅ Continue to use branches and pull requests. When someone on the core team approves a pull request (yours or someone else's), you're welcome to merge it yourself.
* 💚 Commit to setting a good example by following and upholding our [code of conduct](https://cucumber.io/conduct) in your interactions with other collaborators and users.
* 💬 Join the [community Slack channel](http://cucumber.io/support#slack) to meet the rest of the team and make yourself at home.
* ℹ️ Don't feel obliged to help, just do what you can if you have the time and the energy.
* 🙋 Ask if you need anything. We're looking for feedback about how to make the project more welcoming, so please tell us!

On behalf of the Cucumber core team,
Aslak Hellesøy
Creator of Cucumber

**Pull request merged**